### PR TITLE
Add enum system

### PR DESCRIPTION
**Pull request by thenerdie**
Man I'm so uninvolved oh my god

Pull request after 2 weeks = 39 fucking lines of code

### CODE REVIEW

**Comment by Sanjay-B** [comment]
You're merging this into the wrong branch. The merge branch should be ``development``. I've corrected it for you.

**Comment by Sanjay-B** [comment]
You're merging this into the wrong branch. The merge branch should be development. Please change it to the correct branch before I review.

**Comment by thenerdie** [comment]
XDXD LOL ficxed!!121

File: source/libraries/UI/enum.lua [hunk 1]
```diff
@@ -1,5 +1,45 @@
 -- Copyright 2020 - Deviap (deviap.com)
--- Author(s): YOUR NAME, OTHER PERSON'S NAME
+-- Author(s): KISPERAL
 
--- REPLACE WITH ONE LINE DESCRIPTION OF WHAT THIS FILE DOES
-return {}
\ No newline at end of file
+-- PROVIDES A SINGLETON CLASS TO CREATE ENUMS, AND RESOLVE THOSE ENUMS
+
```
Comment (Sanjay-B) [inline source/libraries/UI/enum.lua:5]: You need to document your code. Nothing here is documented. Even if it's straight-forward. 

I need to see two things: 
A). Documentation for what each & every method does. Refer to the other pull requests for the format. 

B). A solid example of this in the works. Something basic but, good enough for me to understand the design & flow of this system.

**Review by Sanjay-B — CHANGES_REQUESTED** [review]

**Comment by thenerdie** [comment]
this should work.... hopefully../.....,,;,,..

File: source/libraries/UI/enum.lua (hunk 1 above)
Comment (thenerdie) [reply source/libraries/UI/enum.lua:5]: I think I did this...

**Review by Sanjay-B — APPROVED** [review]

**Comment by thenerdie** [comment]
jkl; hsdfgkl;d hjfgdfglk jfjlk hdgfhgfhjgdf dgf jlkhgfjdgfl jkhdgfl j ljd jkgdfljk flj jdfgfjk dfgjk hdgfljk hfljk hgflj hkdgfljk hdgfjl gkhdfljd ghkfjldg kfhljdg khfjldg kfhjld gkfhljdfg hkljkdgh fjkl hdfgjld kfgd jlghkfd jlghkfjlhk dgfjkld hgfjld hkgfjl hkdgfjl hkgfjl hkgfjlk hgljk hglj khglj hkljkh gjlhkfgjflhkd gjlhk gjfhlkjhkl jhklgfjlhgfjhl dgfjhkl gjfhkl ghjklf ghjlfk jldhkgfjl ghkf ghjlfk jlghfkjlhfgkl hdjkgfl jhkfglhjgfljhgfljfgljkhfgl jkhgfljkgfl hjkl jkfglj hkl  gf lhjgf lhjkd glfhjk gfhljk gflhjkdgf lhjklhjfklk jhgflj gfl gflhjkgf hjkldgf ljkhdgf lhjkgf lhjkdg flhjkf glhjkd ghjlfd jlgfhd gfjlhd jlhgfhjld kgfhjld kgfdg fhljgj hljkgfd hjlgfkhjl kdfgjldh gfjl hdgjghlkj lhkdgfjlhk dgflhj dkgflgfjlhdg jlhkdgfhjlk djgfhl jdghlfjlfhj ghkld gdfjkgdf jldfg hjlkdgf hjlkjk hl dgfjfgjldgf jlhgf hjlkdgf jhlhjlgf jhkljlhkjl dgfljdfg lhjkdgf jklh gfjlhkdfg jlhkgdf lhjkdgjlf hk

**Pull request merged**